WebSocket/connection.go

```diff
@@ -0,0 +1,69 @@
+package WebSocket
+
```

Comment:
This class is mostly the class provided in the example of gorilla/websocket package. Some minor changes have been made to it